static_src/components/app_container.jsx

```diff
@@ -160,7 +160,7 @@ export default class AppContainer extends React.Component {
               </PageHeader>
             </div>
           </div>
-          <Panel title="Usage and allocation" secondary={ usageSecondary }>
+          <Panel title="App usage and allocation" secondary={ usageSecondary }>
```

Comment:
Yeah, dunno! I added `App` to make it more consistent with the other top panels in the other views, but I didn't think they _all_ needed `App`. But why not? The more the merrier!